features/integration/applications.feature

```diff
@@ -21,8 +21,9 @@ Feature: Applications
       And I sign and submit the transaction, saving the txid. If there is an error it is "".
       And I wait for the transaction to be confirmed.
       And I remember the new application ID.
+      Then I get the account address for the current application and see that it matches the app id's hash
```

Comment:
New cucumber test to check that can calculate an app's account/address from its appid.